### PR TITLE
Spawnables

### PR DESCRIPTION
**Pull request by SerpentSouris26**
Adding unit definitions for logistics

### CODE REVIEW

File: src/logistics/definitions.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@
  * types onto the system so we can build requirements and code completion
  */
 
+import { builtinModules } from 'module'
```
Comment (bitbubbler) [inline src/logistics/definitions.ts]: this appears unused, let's remove it

File: src/logistics/definitions.ts [hunk 2]
```diff
@@ -57,60 +58,435 @@ export type CargoDefinition = BaseCargoDefinition | UnitCargoDefinition
...
+    displayName: 'EWR AN/FPS-117 Radar', // 3 crates E arg[4]
+    id: 'internal-unit-create-fps117-ewr',
+    mass: 1000, 
+    internal: true,
+    type: CargoType.UnitCreate,
+    typeName: CargoTypeName.UH1HCargo,
+    unitTypeName: UnitTypeName.FPS117,
+  },
+  {
+    displayName: 'JTAC HMMWV', // 1 crate I/E arg[5]
```
Comment (bitbubbler) [inline src/logistics/definitions.ts]: >1 crate I/E

this seems like a personal note, it's not clear how it relates to this codebase. We should remove this extra "noise" in the comments here

> arg[5]

this bit can stay, it seems useful

Comment (bitbubbler) [reply src/logistics/definitions.ts]: et. all this file

File: src/logistics/menu.ts [hunk 3]
```diff
@@ -95,6 +106,148 @@ export const internalCargoMenu = groupMenu({
         },
       })
     }
+
+    // Light Units
+    const unitsLightMenu = await addGroupCommandSubMenu({
+      groupName,
+      name: 'Light Units',
+      path: rootPath,
```
Comment (bitbubbler) [inline src/logistics/menu.ts]: Instead of putting these all under the root path, can we nest these under a `Units` menu?

so `rootPath: unitsMenu.path`

Comment (bitbubbler) [reply src/logistics/menu.ts]: et.all to the units sub-menus

**Review by bitbubbler — CHANGES_REQUESTED** [review]
Great work! This is looking really good!

I had a few comments that should be pretty easy to address, nothing major 🥇

File: src/unit.ts [hunk 4]
```diff
@@ -21,10 +21,51 @@ import { countryFrom } from './country'
 
 const { coalition, custom, unit } = services
 
-/** The values of this enum must exactly match dcs in-game unit typeName values */
+/** The values of this enum must exactly match dcs in-ga*me unit typeName values */
```
Comment (bitbubbler) [inline src/unit.ts]:
```suggestion
/** The values of this enum must exactly match dcs in-game unit typeName values */
```

File: src/logistics/definitions.ts (hunk 2 above)
Comment (SerpentSouris26) [reply src/logistics/definitions.ts]: I add those comments for future use of internal/external crates and keep it in mind.
I will remove them.

File: src/logistics/definitions.ts (hunk 1 above)
Comment (SerpentSouris26) [reply src/logistics/definitions.ts]: Done

File: src/logistics/menu.ts [hunk 5]
```diff
@@ -69,13 +79,25 @@ export const internalCargoMenu = groupMenu({
...
+    }
+
+    // Supplies and Utility
+    const suppliesMenu = await addGroupCommandSubMenu({
+      groupName,
+      name: 'Supplies & Utility',
+      path: unitsMenu.path,
+    })
+
+    const unitsPath = suppliesMenu.path
```
Comment (bitbubbler) [inline src/logistics/menu.ts]: This seems confusing. unitsPath = supplies menu path?

**Review by bitbubbler — DISMISSED** [review]

**Review by bitbubbler — APPROVED** [review]

**Pull request merged**